src/printer.js

```diff
@@ -1209,13 +1209,15 @@ function genericPrintNoParens(path, options, print) {
       else parts.push("default:");
 
       if (n.consequent.find(node => node.type !== "EmptyStatement")) {
-        const cons = path.call(
-          function(consequentPath) {
-            return printStatementSequence(consequentPath, options, print);
-          },
-          "consequent"
-        );
-
+        const parent = path.getParentNode();
+        const lastCase = util.getLast(parent.cases);
+        const cons = path.call(consequentPath => {
+          return join(hardline, path.map(p => {
```

Comment:
You are doing `path.call` in order to get `consequentPath` but you're not using it. So either the `path.call` is useless and you should remove it, or there's a problem with the code and `path.map` should be `consequentPath.map` and it happens to be working because path is mutated under the scenes.